src/pages/overview/Overview.vue

```diff
@@ -76,9 +76,10 @@ const infoStore = useInfoStore()
 
 const config = computed(() => ({
   ...infoStore.infoConfig,
+  kongVersion: infoStore.kongVersion,
```

Comment:
Do we need to add `kongVersion` to `config`? I think we can just use `infoStore.kongVersion` in line 82 and line 101.

Reply:
Doesn't seem to make a difference.